simpeg/electromagnetics/natural_source/receivers.py

```diff
@@ -725,69 +498,293 @@ def evalDeriv(self, src, mesh, f, du_dm_v=None, v=None, adjoint=False):
-        :math:`Y_{ij}` corresponding to the data. The data type is specified by
-        the `component` input argument.
-    component : {'real', 'imag'}
-        Admittance data type. For the admittance element :math:`Y_{ij}` specified by the
-        `orientation` input argument, the receiver can be set to compute the following:
-        - 'real': Real component of the admittance (A/V)
-        - 'imag': Imaginary component of the admittance (A/V)
-    locations_e : (n_loc, n_dim) numpy.ndarray
-        Locations where the horizontal electric fields are measured.
-        Must be same size as `locations_h`.
-    locations_h : (n_loc, n_dim) numpy.ndarray
-        Locations where the magnetic fields are measured.
-        Must be same size as `locations_e`.
-    return_complex : bool, optional
-        If ``True`` the complex admittance is evaluation, and the `component` input
-        argument is ignored. Do not use for inversion!
+    locations_r : (n_loc, n_dim) array_like
+        Locations where the roving magnetic fields are measured.
+    locations_b : (n_loc, n_dim) array_like, optional
+        Locations where the base station magnetic fields are measured. Defaults to
+        the same locations as the roving magnetic fields measurements,
+        `locations_r`.
+    orientation : {'xx', 'yx', 'zx', 'zy', 'yy', 'zy'}
+        Specifies the tipper element :math:`T_{ij}` corresponding to the data.
+    component : {'real', 'imag', 'complex'}
+        Tipper data type. For the tipper element :math:`T_{ij}` specified by the `orientation`
+        input argument, the receiver can be set to compute the following:
+        - 'real': Real component of the tipper (unitless)
+        - 'imag': Imaginary component of the tipper (unitless)
+        - 'complex': The complex tipper is returned. Do not use for inversion!
     """
 
+    _loc_names = ("Roving magnetic field", "Base station magnetic field")
+
     def __init__(
         self,
-        locations=None,
-        orientation="xy",
+        locations_r,
+        locations_b=None,
+        orientation="zx",
         component="real",
-        locations_e=None,
-        locations_h=None,
-        return_complex=False,
+        **kwargs,
     ):
-        super().__init__(
-            locations=locations,
-            orientation=orientation,
-            component=component,
-            locations_e=locations_e,
-            locations_h=locations_h,
-            return_complex=return_complex,
+        if locations_b is None:
+            locations_b = locations_r
+        super().__init__(locations1=locations_r, locations2=locations_b, **kwargs)
+        self.orientation = orientation
+        self.component = component
+
+    @property
+    def locations_r(self):
+        """Roving magnetic field measurement locations
+
+        Returns
+        -------
+        numpy.ndarray
+            Roving locations where the magnetic field is measured for all receiver data
```

Comment:
Missing period.

Reply:
@dccowan: one thing you can do on github to help expedite these minor edits is to make `suggestions` on github. In the box where you make a comment, the first icon on the left with the `+ / -` allows you to make a suggested change to that line of code  
![image](https://github.com/simpeg/simpeg/assets/6361812/0e814775-bfcf-476f-a742-0a0d18474357)

It inserts that line in the comment box between the three backticks and you can make the suggested edit there. 
![image](https://github.com/simpeg/simpeg/assets/6361812/23b6a790-395a-4533-80a6-f54b528fd507)

Then, the author / reviewer can simply accept those suggestions and they are committed to the pr (rather than having to make the changes locally). I will do that on this line here. Feel free to give it a go on the other places you noted a missing period below

Reply:
```suggestion
            Roving locations where the magnetic field is measured for all receiver data.
```